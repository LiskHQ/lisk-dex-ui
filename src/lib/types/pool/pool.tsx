export interface LiquidityListInterface {
  token1Id: string;
  token2Id: string;
  feeTier: string;
  pooledtoken1: string;
  pooledtoken2: string;
  poolTokens: string;
  poolShare: string;
  accumulatedFeesToken1: string;
  accumulatedFeesToken2: string;
  totalFees: string;
}

export interface IpriceRange {
  minPrice: string,
  maxPrice: string,
  currentPrice: string  
}

export  interface IdepositAmounts {
  token1: string,
  token2: string,
  amount1: string,
  amount2: string,
  feeTier: string,
}
