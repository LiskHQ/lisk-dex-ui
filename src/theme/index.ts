export { default as theme } from "./theme";
export { default as variables } from "./variables";
