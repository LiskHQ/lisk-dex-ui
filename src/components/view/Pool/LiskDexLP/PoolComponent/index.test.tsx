import React from 'react';
import { ThemeProvider } from '@mui/material';
import { fireEvent, render } from '@testing-library/react';
import { lightTheme } from 'styles/theme';
import { mockPools } from '__mock__/pool.mock';
import { IPoolComponentProps, PoolComponent } from '.';

function renderComponent(props: IPoolComponentProps) {
  return render(
    <ThemeProvider theme={lightTheme} >
      <PoolComponent {...props} />
    </ThemeProvider>
  );
}

describe('PoolComponentt', () => {
  const mockProps: IPoolComponentProps = {
    pool: mockPools[0],
    onIncreaseLiquidity: jest.fn(),
    onRemoveLiquidity: jest.fn(),
  };

  it('checks if the component matches the  snapshot', () => {
    const { container } = renderComponent(mockProps);
    expect(container).toMatchSnapshot();
  });

  it('click on increaseLiquidity', () => {
    const { getByTestId } = renderComponent(mockProps);
    fireEvent.click(getByTestId('increase-liquidity-test'));
    expect(mockProps.onIncreaseLiquidity).toBeCalled();
  });

  it('click on removeLiquidity', () => {
    const { getByTestId } = renderComponent(mockProps);
    fireEvent.click(getByTestId('remove-liquidity-test'));
    expect(mockProps.onRemoveLiquidity).toBeCalled();
  });

  it('checks test-id propoerty works', () => {
    const { getByTestId } = renderComponent({
      ...mockProps,
      'data-testid': 'pool-component-test',
    });

    expect(getByTestId('pool-component-test')).toBeInTheDocument();
  });
});
