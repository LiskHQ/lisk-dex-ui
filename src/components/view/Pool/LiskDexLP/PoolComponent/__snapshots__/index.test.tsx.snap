// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PoolComponentt checks if the component matches the  snapshot 1`] = `
.emotion-0 {
  background-color: #fff;
  color: rgba(0, 0, 0, 0.87);
  -webkit-transition: box-shadow 300ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  transition: box-shadow 300ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  border-radius: 4px;
  box-shadow: 0px 2px 1px -1px rgba(0,0,0,0.2),0px 1px 1px 0px rgba(0,0,0,0.14),0px 1px 3px 0px rgba(0,0,0,0.12);
  position: relative;
  -webkit-transition: margin 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  transition: margin 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  overflow-anchor: none;
  border-radius: 0;
  background: #F2F5F9;
  width: 100%;
  box-shadow: unset;
}

.emotion-0:before {
  position: absolute;
  left: 0;
  top: -1px;
  right: 0;
  height: 1px;
  content: "";
  opacity: 1;
  background-color: rgba(0, 0, 0, 0.12);
  -webkit-transition: opacity 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,background-color 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  transition: opacity 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,background-color 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
}

.emotion-0:first-of-type:before {
  display: none;
}

.emotion-0.Mui-expanded:before {
  opacity: 0;
}

.emotion-0.Mui-expanded:first-of-type {
  margin-top: 0;
}

.emotion-0.Mui-expanded:last-of-type {
  margin-bottom: 0;
}

.emotion-0.Mui-expanded+.emotion-0.Mui-expanded:before {
  display: none;
}

.emotion-0.Mui-disabled {
  background-color: rgba(0, 0, 0, 0.12);
}

.emotion-0:first-of-type {
  border-top-left-radius: 4px;
  border-top-right-radius: 4px;
}

.emotion-0:last-of-type {
  border-bottom-left-radius: 4px;
  border-bottom-right-radius: 4px;
}

@supports (-ms-ime-align: auto) {
  .emotion-0:last-of-type {
    border-bottom-left-radius: 0;
    border-bottom-right-radius: 0;
  }
}

.emotion-0.Mui-expanded {
  margin: 16px 0;
}

.emotion-0 .MuiAccordionSummary-root .MuiAccordionSummary-content {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.emotion-0 .MuiAccordionSummary-root .MuiAccordionSummary-content .pool-summary-token-name {
  margin-left: 0.5rem;
}

.emotion-0 .MuiAccordionSummary-root .MuiAccordionSummary-content .pool-summary-rate {
  background: #D9D8F8;
  padding: 0.25rem 0.5rem;
  border-radius: 0.5rem;
  margin-left: 1rem;
}

.emotion-0 .MuiAccordionSummary-root .MuiAccordionSummary-content .pool-summary-rate p {
  font-size: 0.75rem;
  line-height: 1rem;
}

.emotion-0 .MuiAccordionSummary-expandIconWrapper {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.emotion-0 .MuiAccordionSummary-expandIconWrapper svg {
  color: #6953F4;
  height: 0.75rem;
}

.emotion-0 .MuiAccordionSummary-expandIconWrapper:before {
  content: "Expand";
  color: #6953F4;
  font-size: 0.875rem;
  font-weight: 1.125rem;
  margin-right: 0.25rem;
}

.emotion-0 .MuiAccordionSummary-expandIconWrapper.Mui-expanded {
  -webkit-transform: none;
  -webkit-transform: none;
  -moz-transform: none;
  -ms-transform: none;
  transform: none;
}

.emotion-0 .MuiAccordionSummary-expandIconWrapper.Mui-expanded svg {
  color: #6953F4;
  height: 0.75rem;
  -webkit-transform: rotate(180deg);
  -moz-transform: rotate(180deg);
  -ms-transform: rotate(180deg);
  transform: rotate(180deg);
  -webkit-transform: rotate(180deg);
}

.emotion-0 .MuiAccordionSummary-expandIconWrapper.Mui-expanded:before {
  content: "Collpase";
}

.emotion-0 .MuiAccordionDetails-root {
  border-top: 1px solid #D9D8F8;
  padding: 1.5rem 1rem;
}

.emotion-0 .MuiAccordionDetails-root .pool-details {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

@media (max-width:575.95px) {
  .emotion-0 .MuiAccordionDetails-root .pool-details {
    -webkit-flex-direction: column;
    -ms-flex-direction: column;
    flex-direction: column;
  }

  .emotion-0 .MuiAccordionDetails-root .pool-details .fees-earned {
    margin-top: 2.5rem;
  }
}

.emotion-0 .MuiAccordionDetails-root .pool-details .pool-details-info {
  width: 100%;
}

.emotion-0 .MuiAccordionDetails-root .pool-details .pool-details-info .pool-details-token {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  justify-content: space-between;
  border-radius: 0.5rem;
  margin-top: 1rem;
}

.emotion-0 .MuiAccordionDetails-root .pool-details .pool-details-info .pool-details-token div {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.emotion-0 .MuiAccordionDetails-root .pool-details .pool-details-info .pool-details-token .pool-details-image p {
  margin-left: 0.5rem;
}

.emotion-0 .MuiAccordionDetails-root .pool-details .pool-details-info .pool-details-token .pool-details-amount .MuiBox-root {
  padding: 0.25rem 0.5rem;
  background: #522FEB;
  border-radius: 0.5rem;
  margin-left: 0.5rem;
}

.emotion-0 .MuiAccordionDetails-root .pool-details .pool-details-info .pool-details-token .pool-details-amount .MuiBox-root p {
  font-size: 0.75rem;
  line-height: 1rem;
}

.emotion-0 .MuiAccordionDetails-root .pool-details .pool-details-info.liquidity {
  margin-right: 3rem;
}

.emotion-0 .MuiAccordionDetails-root .pool-details .pool-details-info.liquidity .pool-details-token {
  background: #4738A6;
  padding: 0.5rem 0.75rem;
}

.emotion-0 .MuiAccordionDetails-root .pool-details .pool-details-info.liquidity .pool-details-token p {
  color: #FFFFFF;
}

.emotion-0 .MuiAccordionDetails-root .pool-details .pool-details-info.fees-earned h2 {
  color: #209E1D;
}

.emotion-0 .MuiAccordionDetails-root .pool-details .pool-details-info.fees-earned .pool-details-token {
  padding: 0.5rem 0;
}

.emotion-0 .MuiAccordionDetails-root .pool-details .pool-details-info.fees-earned .pool-details-amount span {
  margin-left: 0.25rem;
}

.emotion-0 .MuiAccordionDetails-root .pool-details .pool-detials-fees-earned {
  width: 100%;
}

.emotion-0 .MuiAccordionDetails-root .pool-share {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  justify-content: space-between;
  margin-top: 2rem;
}

.emotion-0 .MuiAccordionDetails-root .pool-share .pool-share-title {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.emotion-0 .MuiAccordionDetails-root .pool-share .pool-share-title svg {
  width: 1rem;
  height: 1rem;
  margin-right: 0.5rem;
}

.emotion-0 .MuiAccordionDetails-root .pool-share .pool-share-title svg path {
  stroke: #282828;
}

.emotion-0 .MuiAccordionDetails-root .pool-buttons {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin-top: 3rem;
}

@media (max-width:575.95px) {
  .emotion-0 .MuiAccordionDetails-root .pool-buttons {
    -webkit-flex-direction: column;
    -ms-flex-direction: column;
    flex-direction: column;
  }

  .emotion-0 .MuiAccordionDetails-root .pool-buttons .remove-liquidity {
    margin-bottom: 1rem;
  }
}

.emotion-0 .MuiAccordionDetails-root .pool-buttons .remove-liquidity {
  margin-right: 1.5rem;
}

.emotion-1 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  position: relative;
  box-sizing: border-box;
  -webkit-tap-highlight-color: transparent;
  background-color: transparent;
  outline: 0;
  border: 0;
  margin: 0;
  border-radius: 0;
  padding: 0;
  cursor: pointer;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  vertical-align: middle;
  -moz-appearance: none;
  -webkit-appearance: none;
  -webkit-text-decoration: none;
  text-decoration: none;
  color: inherit;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  min-height: 48px;
  padding: 0px 16px;
  -webkit-transition: min-height 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,background-color 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  transition: min-height 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,background-color 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
}

.emotion-1::-moz-focus-inner {
  border-style: none;
}

.emotion-1.Mui-disabled {
  pointer-events: none;
  cursor: default;
}

@media print {
  .emotion-1 {
    -webkit-print-color-adjust: exact;
    color-adjust: exact;
  }
}

.emotion-1.Mui-focusVisible {
  background-color: rgba(0, 0, 0, 0.12);
}

.emotion-1.Mui-disabled {
  opacity: 0.38;
}

.emotion-1:hover:not(.Mui-disabled) {
  cursor: pointer;
}

.emotion-1.Mui-expanded {
  min-height: 64px;
}

.emotion-2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
  margin: 12px 0;
  -webkit-transition: margin 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  transition: margin 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
}

.emotion-2.Mui-expanded {
  margin: 20px 0;
}

.emotion-3 {
  margin: 0;
  font-size: 1rem;
  line-height: 1.375rem;
  font-weight: 500;
  font-family: "Roboto","Helvetica","Arial",sans-serif;
  letter-spacing: 0.00938em;
}

.emotion-5 {
  margin: 0;
  font-size: 0.875rem;
  line-height: 1.125rem;
  font-weight: 400;
  font-family: "Roboto","Helvetica","Arial",sans-serif;
  letter-spacing: 0.01071em;
}

.emotion-6 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  color: rgba(0, 0, 0, 0.54);
  -webkit-transform: rotate(0deg);
  -moz-transform: rotate(0deg);
  -ms-transform: rotate(0deg);
  transform: rotate(0deg);
  -webkit-transition: -webkit-transform 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  transition: transform 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
}

.emotion-6.Mui-expanded {
  -webkit-transform: rotate(180deg);
  -moz-transform: rotate(180deg);
  -ms-transform: rotate(180deg);
  transform: rotate(180deg);
}

.emotion-7 {
  height: 0;
  overflow: hidden;
  -webkit-transition: height 300ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  transition: height 300ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  visibility: hidden;
}

.emotion-8 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  width: 100%;
}

.emotion-9 {
  width: 100%;
}

.emotion-10 {
  padding: 8px 16px 16px;
}

.emotion-14 {
  margin: 0;
  font-size: 2rem;
  font-weight: 600;
  font-family: "Roboto","Helvetica","Arial",sans-serif;
  line-height: 1.2;
  letter-spacing: -0.00833em;
}

.emotion-37 {
  margin: 0;
  font-size: 0.75rem;
  line-height: 1rem;
  font-weight: 500;
  font-family: "Roboto","Helvetica","Arial",sans-serif;
  letter-spacing: 0.03333em;
}

.emotion-50 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  position: relative;
  box-sizing: border-box;
  -webkit-tap-highlight-color: transparent;
  background-color: transparent;
  outline: 0;
  border: 0;
  margin: 0;
  border-radius: 0;
  padding: 0;
  cursor: pointer;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  vertical-align: middle;
  -moz-appearance: none;
  -webkit-appearance: none;
  -webkit-text-decoration: none;
  text-decoration: none;
  color: inherit;
  font-family: "Roboto","Helvetica","Arial",sans-serif;
  font-weight: 500;
  font-size: 0.875rem;
  line-height: 1.75;
  letter-spacing: 0.02857em;
  text-transform: uppercase;
  min-width: 64px;
  padding: 5px 15px;
  border-radius: 4px;
  -webkit-transition: background-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,box-shadow 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,border-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  transition: background-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,box-shadow 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,border-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  border: 1px solid rgba(25, 118, 210, 0.5);
  color: #1976d2;
  width: 100%;
  color: #FF4557;
  padding: 0.75rem;
  text-transform: capitalize;
}

.emotion-50::-moz-focus-inner {
  border-style: none;
}

.emotion-50.Mui-disabled {
  pointer-events: none;
  cursor: default;
}

@media print {
  .emotion-50 {
    -webkit-print-color-adjust: exact;
    color-adjust: exact;
  }
}

.emotion-50:hover {
  -webkit-text-decoration: none;
  text-decoration: none;
  background-color: rgba(25, 118, 210, 0.04);
  border: 1px solid #1976d2;
}

@media (hover: none) {
  .emotion-50:hover {
    background-color: transparent;
  }
}

.emotion-50.Mui-disabled {
  color: rgba(0, 0, 0, 0.26);
  border: 1px solid rgba(0, 0, 0, 0.12);
}

.emotion-50.MuiButton-outlined {
  border-color: #FF4557;
}

.emotion-50.MuiButton-contained {
  background: #6953F4;
  color: #FFFFFF;
}

.emotion-50.Mui-disabled {
  opacity: 0.7;
}

.emotion-52 {
  overflow: hidden;
  pointer-events: none;
  position: absolute;
  z-index: 0;
  top: 0;
  right: 0;
  bottom: 0;
  left: 0;
  border-radius: inherit;
}

.emotion-53 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  position: relative;
  box-sizing: border-box;
  -webkit-tap-highlight-color: transparent;
  background-color: transparent;
  outline: 0;
  border: 0;
  margin: 0;
  border-radius: 0;
  padding: 0;
  cursor: pointer;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  vertical-align: middle;
  -moz-appearance: none;
  -webkit-appearance: none;
  -webkit-text-decoration: none;
  text-decoration: none;
  color: inherit;
  font-family: "Roboto","Helvetica","Arial",sans-serif;
  font-weight: 500;
  font-size: 0.875rem;
  line-height: 1.75;
  letter-spacing: 0.02857em;
  text-transform: uppercase;
  min-width: 64px;
  padding: 6px 16px;
  border-radius: 4px;
  -webkit-transition: background-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,box-shadow 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,border-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  transition: background-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,box-shadow 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,border-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  color: #fff;
  background-color: #1976d2;
  box-shadow: 0px 3px 1px -2px rgba(0,0,0,0.2),0px 2px 2px 0px rgba(0,0,0,0.14),0px 1px 5px 0px rgba(0,0,0,0.12);
  width: 100%;
  color: #FF4557;
  padding: 0.75rem;
  text-transform: capitalize;
}

.emotion-53::-moz-focus-inner {
  border-style: none;
}

.emotion-53.Mui-disabled {
  pointer-events: none;
  cursor: default;
}

@media print {
  .emotion-53 {
    -webkit-print-color-adjust: exact;
    color-adjust: exact;
  }
}

.emotion-53:hover {
  -webkit-text-decoration: none;
  text-decoration: none;
  background-color: #1565c0;
  box-shadow: 0px 2px 4px -1px rgba(0,0,0,0.2),0px 4px 5px 0px rgba(0,0,0,0.14),0px 1px 10px 0px rgba(0,0,0,0.12);
}

@media (hover: none) {
  .emotion-53:hover {
    background-color: #1976d2;
  }
}

.emotion-53:active {
  box-shadow: 0px 5px 5px -3px rgba(0,0,0,0.2),0px 8px 10px 1px rgba(0,0,0,0.14),0px 3px 14px 2px rgba(0,0,0,0.12);
}

.emotion-53.Mui-focusVisible {
  box-shadow: 0px 3px 5px -1px rgba(0,0,0,0.2),0px 6px 10px 0px rgba(0,0,0,0.14),0px 1px 18px 0px rgba(0,0,0,0.12);
}

.emotion-53.Mui-disabled {
  color: rgba(0, 0, 0, 0.26);
  box-shadow: none;
  background-color: rgba(0, 0, 0, 0.12);
}

.emotion-53.MuiButton-outlined {
  border-color: #FF4557;
}

.emotion-53.MuiButton-contained {
  background: #6953F4;
  color: #FFFFFF;
}

.emotion-53.Mui-disabled {
  opacity: 0.7;
}

<div>
  <div
    class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiAccordion-root MuiAccordion-rounded MuiAccordion-gutters emotion-0"
  >
    <div
      aria-expanded="false"
      class="MuiButtonBase-root MuiAccordionSummary-root MuiAccordionSummary-gutters emotion-1"
      role="button"
      tabindex="0"
    >
      <div
        class="MuiAccordionSummary-content MuiAccordionSummary-contentGutters emotion-2"
      >
        <img
          alt="DEX"
          height="24"
          src="https://raw.githubusercontent.com/LiskHQ/app-registry/main/devnet/Dex/images/tokens/dex.png"
          style="border-radius: 100%;"
          width="24"
        />
        <img
          alt="ENVT"
          height="24"
          src="https://raw.githubusercontent.com/LiskHQ/app-registry/main/devnet/Enevti/images/tokens/enevti.png"
          style="border-radius: 100%;"
          width="24"
        />
        <p
          class="MuiTypography-root MuiTypography-body1 pool-summary-token-name emotion-3"
        >
          DEX
           / 
          ENVT
        </p>
        <div
          class="pool-summary-rate MuiBox-root emotion-4"
        >
          <p
            class="MuiTypography-root MuiTypography-body2 emotion-5"
          >
            0.05
          </p>
        </div>
      </div>
      <div
        class="MuiAccordionSummary-expandIconWrapper emotion-6"
      >
        <svg
          aria-hidden="true"
          class="svg-inline--fa fa-chevron-down "
          data-icon="chevron-down"
          data-prefix="fas"
          focusable="false"
          role="img"
          viewBox="0 0 512 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M233.4 406.6c12.5 12.5 32.8 12.5 45.3 0l192-192c12.5-12.5 12.5-32.8 0-45.3s-32.8-12.5-45.3 0L256 338.7 86.6 169.4c-12.5-12.5-32.8-12.5-45.3 0s-12.5 32.8 0 45.3l192 192z"
            fill="currentColor"
          />
        </svg>
      </div>
    </div>
    <div
      class="MuiCollapse-root MuiCollapse-vertical MuiCollapse-hidden emotion-7"
      style="min-height: 0px;"
    >
      <div
        class="MuiCollapse-wrapper MuiCollapse-vertical emotion-8"
      >
        <div
          class="MuiCollapse-wrapperInner MuiCollapse-vertical emotion-9"
        >
          <div
            class="MuiAccordion-region"
            role="region"
          >
            <div
              class="MuiAccordionDetails-root emotion-10"
            >
              <div
                class="pool-details MuiBox-root emotion-4"
              >
                <div
                  class="pool-details-info liquidity MuiBox-root emotion-4"
                >
                  <p
                    class="MuiTypography-root MuiTypography-body1 emotion-3"
                  >
                    Liquidity
                  </p>
                  <h2
                    class="MuiTypography-root MuiTypography-h2 emotion-14"
                  >
                    $2,523
                  </h2>
                  <div
                    class="pool-details-token token1 MuiBox-root emotion-4"
                  >
                    <div
                      class="pool-details-image MuiBox-root emotion-4"
                    >
                      <img
                        alt="DEX"
                        height="24"
                        src="https://raw.githubusercontent.com/LiskHQ/app-registry/main/devnet/Dex/images/tokens/dex.png"
                        style="border-radius: 100%;"
                        width="24"
                      />
                      <p
                        class="MuiTypography-root MuiTypography-body2 emotion-5"
                      >
                        DEX
                      </p>
                    </div>
                    <div
                      class="pool-details-amount MuiBox-root emotion-4"
                    >
                      <p
                        class="MuiTypography-root MuiTypography-body2 emotion-5"
                      >
                        1862.000000000000
                      </p>
                      <div
                        class="MuiBox-root emotion-4"
                      >
                        <p
                          class="MuiTypography-root MuiTypography-body2 emotion-5"
                        >
                          55%
                        </p>
                      </div>
                    </div>
                  </div>
                  <div
                    class="pool-details-token token2 MuiBox-root emotion-4"
                  >
                    <div
                      class="pool-details-image MuiBox-root emotion-4"
                    >
                      <img
                        alt="ENVT"
                        height="24"
                        src="https://raw.githubusercontent.com/LiskHQ/app-registry/main/devnet/Enevti/images/tokens/enevti.png"
                        style="border-radius: 100%;"
                        width="24"
                      />
                      <p
                        class="MuiTypography-root MuiTypography-body2 emotion-5"
                      >
                        ENVT
                      </p>
                    </div>
                    <div
                      class="pool-details-amount MuiBox-root emotion-4"
                    >
                      <p
                        class="MuiTypography-root MuiTypography-body2 emotion-5"
                      >
                        0.04500000000000000
                      </p>
                      <div
                        class="MuiBox-root emotion-4"
                      >
                        <p
                          class="MuiTypography-root MuiTypography-body2 emotion-5"
                        >
                          55%
                        </p>
                      </div>
                    </div>
                  </div>
                </div>
                <div
                  class="pool-details-info fees-earned MuiBox-root emotion-4"
                >
                  <p
                    class="MuiTypography-root MuiTypography-body1 emotion-3"
                  >
                    Fees earned
                  </p>
                  <h2
                    class="MuiTypography-root MuiTypography-h2 emotion-14"
                  >
                    $105.76
                  </h2>
                  <div
                    class="pool-details-token token1 MuiBox-root emotion-4"
                  >
                    <div
                      class="pool-details-image MuiBox-root emotion-4"
                    >
                      <img
                        alt="DEX"
                        height="20"
                        src="https://raw.githubusercontent.com/LiskHQ/app-registry/main/devnet/Dex/images/tokens/dex.png"
                        style="border-radius: 100%;"
                        width="20"
                      />
                      <p
                        class="MuiTypography-root MuiTypography-body2 emotion-5"
                      >
                        DEX
                      </p>
                    </div>
                    <div
                      class="pool-details-amount MuiBox-root emotion-4"
                    >
                      <p
                        class="MuiTypography-root MuiTypography-body2 emotion-5"
                      >
                        0.034
                      </p>
                      <span
                        class="MuiTypography-root MuiTypography-caption emotion-37"
                      >
                        ($52.42)
                      </span>
                    </div>
                  </div>
                  <div
                    class="pool-details-token token2 MuiBox-root emotion-4"
                  >
                    <div
                      class="pool-details-image MuiBox-root emotion-4"
                    >
                      <img
                        alt="ENVT"
                        height="20"
                        src="https://raw.githubusercontent.com/LiskHQ/app-registry/main/devnet/Enevti/images/tokens/enevti.png"
                        style="border-radius: 100%;"
                        width="20"
                      />
                      <p
                        class="MuiTypography-root MuiTypography-body2 emotion-5"
                      >
                        ENVT
                      </p>
                    </div>
                    <div
                      class="pool-details-amount MuiBox-root emotion-4"
                    >
                      <p
                        class="MuiTypography-root MuiTypography-body2 emotion-5"
                      >
                        49.251
                      </p>
                      <span
                        class="MuiTypography-root MuiTypography-caption emotion-37"
                      >
                        ($53.98)
                      </span>
                    </div>
                  </div>
                </div>
              </div>
              <div
                class="pool-share MuiBox-root emotion-4"
              >
                <div
                  class="pool-share-title MuiBox-root emotion-4"
                >
                  <svg
                    fill="none"
                    height="20"
                    viewBox="0 0 20 20"
                    width="20"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <g
                      clip-path="url(#clip0_581_45332)"
                    >
                      <path
                        d="M17.6766 13.2417C17.1465 14.4955 16.3173 15.6002 15.2615 16.4595C14.2058 17.3187 12.9556 17.9063 11.6203 18.1707C10.2851 18.4352 8.90532 18.3685 7.60176 17.9766C6.2982 17.5846 5.11049 16.8793 4.14249 15.9223C3.17448 14.9653 2.45565 13.7857 2.04883 12.4867C1.64202 11.1877 1.55961 9.8088 1.8088 8.47059C2.058 7.13238 2.63122 5.87559 3.47834 4.81009C4.32547 3.74459 5.42071 2.90283 6.66831 2.3584"
                        stroke="#3D3D3D"
                        stroke-linecap="round"
                        stroke-linejoin="round"
                        stroke-width="1.67"
                      />
                      <path
                        d="M18.3333 9.99984C18.3333 8.90549 18.1178 7.82185 17.699 6.81081C17.2802 5.79976 16.6664 4.8811 15.8926 4.10728C15.1187 3.33346 14.2001 2.71963 13.189 2.30084C12.178 1.88205 11.0943 1.6665 10 1.6665V9.99984H18.3333Z"
                        stroke="#3D3D3D"
                        stroke-linecap="round"
                        stroke-linejoin="round"
                        stroke-width="1.67"
                      />
                    </g>
                    <defs>
                      <clippath
                        id="clip0_581_45332"
                      >
                        <rect
                          fill="white"
                          height="20"
                          width="20"
                        />
                      </clippath>
                    </defs>
                  </svg>
                  <p
                    class="MuiTypography-root MuiTypography-body2 emotion-5"
                  >
                    Your pool share:
                  </p>
                </div>
                <div
                  class="pool-share-percent MuiBox-root emotion-4"
                >
                  <p
                    class="MuiTypography-root MuiTypography-body2 emotion-5"
                  >
                    0.085%
                  </p>
                </div>
              </div>
              <div
                class="pool-buttons MuiBox-root emotion-4"
              >
                <button
                  class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium remove-liquidity emotion-50"
                  data-testid="remove-liquidity-test"
                  tabindex="0"
                  type="button"
                >
                  <p
                    class="MuiTypography-root MuiTypography-body1 emotion-3"
                  >
                    - Remove Liquidity
                  </p>
                  <span
                    class="MuiTouchRipple-root emotion-52"
                  />
                </button>
                <button
                  class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium emotion-53"
                  data-testid="increase-liquidity-test"
                  tabindex="0"
                  type="button"
                >
                  <p
                    class="MuiTypography-root MuiTypography-body1 emotion-3"
                  >
                    + Increase Liquidity
                  </p>
                  <span
                    class="MuiTouchRipple-root emotion-52"
                  />
                </button>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
